frontend/lib/parsing/parser-error-classes-list.cpp

```diff
@@ -248,6 +248,13 @@ void ErrorCantApplyPrivate::write(ErrorWriterBase& wr) const {
   wr.code(node);
 }
 
+void ErrorWhenAfterOtherwise::write(ErrorWriterBase& wr) const {
+  auto node = std::get<const uast::Select*>(info);
+  wr.heading(kind_, type_, node, "otherwise clause must follow all when clauses.");
+  wr.message("The following select statement has improperly ordered blocks:");
+  wr.code(node);
```

Comment:
When writing language keywords like `otherwise` or `select` or `when`, we try to quote them: `'otherwise' clause must follow all 'when' clauses`.